docs/administration/cluster/loadbalancer/reverse_proxies.md

```diff
@@ -0,0 +1,279 @@
+# How to Configure Rundeck Behind a Reverse Proxy Server
```

Comment:
This likely needs an entry in `sidebar-menus.js` to show up in navigation on the left.